templates/_base/footer.html

```diff
@@ -1,32 +1,226 @@
 {% block footer %}
-<hr class="u-no-margin--bottom">
-<section class="p-strip u-no-margin--top">
-  <div class="row">
-    <p>&copy; {{ now("%Y") }} Canonical Ltd. Ubuntu および Canonical は、Canonical Ltd の登録商標です。</p>
-    <nav class="u-no-margin--top">
-      <ul class="p-inline-list--middot">
-        <li class="p-inline-list__item">
-          <a class="p-link--soft" accesskey="9" href="https://ubuntu.com/legal/terms-and-policies/privacy-policy">Cookie policy</a>
-        </li>
-        <li class="p-inline-list__item">
-          <a class="p-link--soft js-revoke-cookie-manager" href="">Manage your tracker settings</a>
-        </li>
-        <li class="p-inline-list__item">
-          <a class="p-link--soft" href="https://github.com/canonical-websites/jp.ubuntu.com/issues/new" id="report-a-bug">Report a bug on this site</a>
-        </li>
-      </ul>
-
-      <script>
-        /* Add the page to the report a bug link */
-        var bugLink = document.querySelector('#report-a-bug');
-        bugLink.href += '?body=%0a%0a%0a---%0a*Reported%20from:%20' + location.href + '*';
-      </script>
-
-      <span class="u-off-screen">
-        <a href="#">Go to the top of the page</a>
-      </span>
-    </nav>
-  </div>
+<section class="p-strip p-footer u-no-margin--top">
+  <section class="p-section">
+    <div class="row">
+      <div class="col-3 col-medium-1">製品</div>
+      <div class="col-9 col-medium-5">
+        <ul class="p-inline-list">
+          <li class="p-inline-list__item">
+            <a href="/openstack">OpenStack</a>
+          </li>
+          <li class="p-inline-list__item">
+            <a href="/kubernetes">Kubernetes</a>
+          </li>
+          <li class="p-inline-list__item">
+            <a href="https://canonical.com/microcloud">MicroCloud</a>
+          </li>
+          <li class="p-inline-list__item">
+            <a href="/enterprise-support">商用サポート</a>
+          </li>
+          <li class="p-inline-list__item">
+            <a href="/certified">認証済みH/W
```

Comment:
looks like this page doesn't exist yet. Shouldn't we remove this link then?

Reply:
Will this be launched together with https://warthogs.atlassian.net/browse/WD-10741 ?

Reply:
@akbarkz Thanks for the catch, I'll remove the /certified links for now, will add it back once /certified is launched.
@juanruitina No, it will be launched separately.